src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1522,6 +1558,10 @@ internal bool CanAddLookupSymbolInfo(Symbol symbol, LookupOptions options, Looku
             {
                 return false;
             }
+            else if (!IsInScopeOfAssociatedSyntaxTree(symbol))
```

Comment:
>IsInScopeOfAssociatedSyntaxTree [](http://example.com/codeflow?start=22&length=31)

Isn't this covered by the previous `else if`? Specifically, won't `this.IsAccessible(symbol, ...)` handle this?

Reply:
No, we're treating this as a "parallel" concept to accessibility. So IsAccessible, IsAtLeastAsVisibleAs, etc. don't consider file types.